packages/next-templates/src/app/globals.css

```diff
@@ -280,6 +285,18 @@ utrecht-alert
   --utrecht-accordion-panel-padding-inline-end: var(--voorbeeld-space-inline-pig);
 }
 
+.voorbeeld-unordered-list-cards {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-gap: 1rem;
+  list-style: none;
+  padding-inline-start: 0;
+}
+
+.voorbeeld-container-card {
+  border-block-start: 0.15em solid var(--voorbeeld-color-gray-200);
```

Comment:
Ik stel voor om een border om de hele card te doen, dat is een iets minder eigenzinnig design — dat past beter bij gemeente Voorbeeld.

Reply:
Als je helemaal los wilt gaan, dan doe je voor [`prefers-contrast: more`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast) een donkerdere border grijs